### ISSUE TITLE
Add configuration for Savon timeout

### ISSUE THREAD

**Issue by ndnenkov** [yolk/valvat#54]
VIES has it's ups and downs. There are periods where it simply takes too long to respond for certain uses. Right now the only way to make a request with custom timeout is something like:

```ruby
client   = Savon.client open_timeout: 10,
                        read_timeout: 10,
                        wsdl: Valvat::Lookup::VIES_WSDL_URL,
                        log: false
response = Valvat::Lookup::Request.new(Valvat.new vat).perform client
```

**Comment by yolk** [comment]
Hi Chris,

the current master branch includes (not yet released)  changes that allow to pass in a `savon` option key to configure any aspect of the savon client.
Hope this helps.

Thanks,
Sebastian.

**Comment by yolk** [comment]
Just released version 1.0.0 includes this fix.
